woocommerce/payment-gateway/class-sv-wc-payment-gateway-my-payment-methods.php

```diff
@@ -857,22 +854,28 @@ protected function get_payment_method_title_html( SV_WC_Payment_Gateway_Payment_
 	 *
 	 * @since 5.1.0
 	 *
-	 * @param SV_WC_Payment_Gateway_Payment_Token $token token object
+	 * @param boolean $is_default true if the token is the default token
+	 * @param SV_WC_Payment_Gateway_Payment_Token|null $token FW token object, only set if the token is a FW token
 	 * @return string
 	 */
-	protected function get_payment_method_default_html( SV_WC_Payment_Gateway_Payment_Token $token ) {
+	protected function get_payment_method_default_html( $is_default, SV_WC_Payment_Gateway_Payment_Token $token = null ) {
```

Comment:
@deivamagalhaes assuming our min WP/WC PHP version does not support type casting here?  If it does this bool really should be type casted and set to a default of false.

Reply:
We support PHP 5.6, so yes, no scalar type declarations for us. I can still set the default though, if you think that would be helpful.

Reply:
Yeah I think we should default to that on a bool parameter.  Can easily have someone do a strict comparison later and be unaware.

Reply:
Ah my spelling sucks.. lets try this again: Yeah I think we should default to that on a bool parameter for readability. And it saves a step keeping things clear in the future where we do upgrade PHP versions.